packages/console/src/components/TextInput/index.module.scss

```diff
@@ -31,6 +31,9 @@
     color: var(--color-component-text);
     font: var(--font-body-medium);
     padding: 0;
+    // Our font size and line height pair (14px/20px) will cause a "blank top space" in inline display.
+    // need to set display to block to keep the whole input box height to 36px.
+    display: block;
```

Comment:
How about overriding the line-height to 36px? Changing `display` type would probably cause unexpected behaviors in some edge cases

Reply:
updated, PTAL